TextureExtraction tool/Data/ScanBase.cs

```diff
@@ -27,12 +29,34 @@ public abstract class ScanBase
 
         public class Options
         {
+            static NameValueCollection config;
+            public static NameValueCollection Config => config = config ?? ConfigurationManager.AppSettings;
+            static bool? useConfig = null;
+            public static bool UseConfig = (bool)(useConfig = useConfig ?? Config.HasKeys() && bool.TryParse(Config.Get("UseConfig"), out bool value) && value);
 #if DEBUG
             public ParallelOptions Parallel = new ParallelOptions() { MaxDegreeOfParallelism = 1 };
 #else
-            public ParallelOptions Parallel = new ParallelOptions() { MaxDegreeOfParallelism = 4 };
+            public ParallelOptions Parallel = new ParallelOptions() { MaxDegreeOfParallelism = -1 };
```

Comment:
You would think that -1 is faster, but it's not.
I have tested again some games and it takes 20-40% longer and consumes more memory and cpu.